backend/src/common/enums/exception/exception-message.enum.ts

```diff
@@ -39,6 +39,7 @@ const ExceptionMessage = {
   SSH_KEY_NOT_FOUND: 'Ssh key not found',
   NOTHING_TO_UPDATE: 'Nothing to update',
   SPACE_EXISTS: 'Space with this name already exists',
+  REQUIRED_PERMISSION: 'Permissions are required to access',
```

Comment:
I think it is more clear if write "You don't have required permissions to access."
Question for coaches.